src/map/clif.h

```diff
@@ -1209,7 +1209,8 @@ struct clif_interface {
 	void (*useskill) (struct block_list* bl, int src_id, int dst_id, int dst_x, int dst_y, uint16 skill_id, uint16 skill_lv, int casttime);
 	void (*produce_effect) (struct map_session_data* sd,int flag,int nameid);
 	void (*devotion) (struct block_list *src, struct map_session_data *tsd);
-	void (*spiritball) (struct block_list *bl, enum spirit_ball_types spirit, enum send_target target);
+	void (*soulball) (struct block_list *bl, int soulballs, enum send_target target);
+	void (*spiritball) (struct block_list *bl, int spiritballs, enum send_target target);
```

Comment:
Can you change this signature (i.e. rename) so that it becomes incompatible with the previous version?

The goal is to avoid old custom code that was calling `clif->spiritball()` to compile without errors or warnings, and silently cause unexpected behavior since the meaning of the second argument has changed in an incompatible way

Reply:
I thought the change from enum to int would be enough, but is not... updated by changing both to plural (`clif->spiritballs` / `clif->soulball`).

Thanks!